sdk/appcenter-auth/src/test/java/com/microsoft/appcenter/auth/AuthTest.java

```diff
@@ -96,6 +96,57 @@
     @Captor
     private ArgumentCaptor<Map<String, String>> mHeadersCaptor;
 
```

Comment:
Looks like whitespaces are changed on every single line, can this be fixed?

Reply:
If you are referring below methods, it is moved to the top due to private static methods order.

Reply:
Looks like last commits fixed it somehow.